Packs/ApiModules/ReleaseNotes/2_2_0.md

```diff
@@ -3,3 +3,6 @@
 ##### JSONFeedApiModule
 - Added support for grid fields hierarchy.
 - Added option for flat JSON response.
+
+##### HTTPFeedApiModule
```

Comment:
revert this change, there shouldn't be release notes for API modules

Reply:
Done